test/functional/test_runner.py

```diff
@@ -108,6 +108,7 @@
     'rpcnamedargs.py',
     'listsinceblock.py',
     'p2p-leaktests.py',
+    'import-abort-rescan.py',           # ~17s
```

Comment:
I think we should remove the time hint, otherwise this will become "the standard" and, with that, the "keep-it-updated" problem will follow.